### ISSUE TITLE
Remove/replace label for current stage when a fork's branch is deleted without being merged

### ISSUE THREAD

**Issue by misskniss** [codetree/feedback#291]
depends on #290 

Actual Behavior: 
When an issue is automatically moved from the "To-Do" stage to the custom stage "In Progress" based on a new branch in a developer's fork, it gets the In Progress label as expected. However, if that remote branch is then deleted by the developer without being merged into the upstream repo (or it's PR is closed instead of merged) the issue remains in the "In Progress" stage and retains the In Progress label even though it is technically  a ToDo item again. 

Expected Behavior: 
Reverse the rule for the stage transition when a remote branch is deleted (without being merged) from a developer's fork. i.e.: Remove the In Progress label and replace the ToDo label so the stages accurately reflect that this issue was being worked on but now is no longer In Progress. 

In the case where the developer is cleaning up their fork's  branches after their branch has been Pr'ed and merge into the upstream repo - we would NOT expect to see the issue get labelled as ToDo since it is Completed and Merged.
